agent/commands/run.go

```diff
@@ -98,7 +107,7 @@ func run(ctx context.Context, cfg *config.Config, configFilepath string) {
 	connectionChecker := connectionchecker.New(&cfg.Paths)
 	defaultsFileParser := defaultsfile.New()
 	v := versioner.New(&versioner.RealExecFunctions{})
-	client := client.New(cfg, supervisor, connectionChecker, v, defaultsFileParser)
+	client := client.New(cfg, supervisor, connectionChecker, v, defaultsFileParser, cs)
 	localServer := agentlocal.NewServer(cfg, supervisor, client, configFilepath)
```

Comment:
I think we can pass `cs` to Server and remove `GetConnectionUpTime` from the client. What do you think?